src/Core/Prefab/GlobeView.js

```diff
@@ -389,8 +411,13 @@ const screen = new THREE.Vector2();
 const pickWorldPosition = new THREE.Vector3();
 const ray = new THREE.Ray();
 const direction = new THREE.Vector3();
-GlobeView.prototype.getPickingPositionFromDepth = function getPickingPositionFromDepth(mouse) {
-    const dim = this.mainLoop.gfxEngine.getWindowSize();
+GlobeView.prototype.getPickingPositionFromDepth = function getPickingPositionFromDepth(mouse, readFullBuffer) {
```

Comment:
What's the usage of this `readFullBuffer` parameter?

Reply:
removed